src/test/java/duke/EventTest.java

```diff
@@ -0,0 +1,24 @@
+package duke;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EventTest {
+
+    @Test
+    public void dummyTest(){
+        assertEquals(2, 2);
+    }
+
+    /**
+     * Tests if the event toString method prints correctly.
+     */
+    @Test
+    public void eventToStringTest(){
```

Comment:
Perhaps the test method could be renamed such that it follows the format given in the coding standard `featureUnderTest_testScenario_expectedBehavior()`?